release/models/interfaces/openconfig-if-common.yang

```diff
@@ -0,0 +1,301 @@
+submodule openconfig-if-common {
```

Comment:
What's the motivation to create a `submodule` here? We've done it in cases where the module is very large (e.g., BGP), but in this case it's relatively small numbers of shared leaves - and it doesn't really simplify the imports.

Reply:
removed the submodule and moved grouping into openconfig-interfaces.yang.